api-admin/src/main/java/bio/terra/pearl/api/admin/service/study/StudyExtService.java

```diff
@@ -66,9 +80,43 @@ public Study create(String portalShortcode, StudyCreationDto study, AdminUser op
             .build();
     newStudy = studyService.create(newStudy);
     portalStudyService.create(portal.getId(), newStudy.getId());
+
+    if (Objects.nonNull(study.getTemplate())) {
+      fillInWithTemplate(portalShortcode, newStudy, study.getTemplate());
+    }
+
     return newStudy;
   }
 
+  private void fillInWithTemplate(
+      String portalShortcode, Study newStudy, StudyCreationDto.StudyTemplate studyTemplate) {
+    String filename;
+
+    switch (studyTemplate) {
+      default -> {
+        filename = "basic_study.json";
+      }
+    }
+
+    PortalPopulateContext config =
+        new PortalPopulateContext(
+            "templates/" + filename, portalShortcode, null, new HashMap<>(), false);
+
+    StudyPopDto studyPopDto;
+
+    try {
+      String fileContents = filePopulateService.readFile(filename, config);
+      studyPopDto = studyPopulator.readValue(fileContents);
+
+      studyPopDto.setShortcode(newStudy.getShortcode());
+      studyPopDto.setName(newStudy.getName());
+
+      studyPopulator.createPreserveExisting(newStudy, studyPopDto, config);
```

Comment:
congrats on working through the very wonky populator interfaces.  I think you want to use `studyPopulator.populateFromDto()` here though, as that picks up an extra preProcessing step (which is a no-op for studies currently, but could have code later).    I think createPreserveExisting is public from a leftover time when Populator was an interface.  createPreserveExisting should be protected (although changing it is outside of this PR)